plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaConfig.java

```diff
@@ -146,10 +158,49 @@ public boolean isTimestampUpperBoundPushDownEnabled()
     }
 
     @Config("kafka.timestamp-upper-bound-force-push-down-enabled")
-    @ConfigDescription("timestamp upper bound force pushing down enabled")
+    @ConfigDescription("Timestamp upper bound force pushing down enabled")
     public KafkaConfig setTimestampUpperBoundPushDownEnabled(boolean timestampUpperBoundPushDownEnabled)
     {
         this.timestampUpperBoundPushDownEnabled = timestampUpperBoundPushDownEnabled;
         return this;
     }
+
+    @NotNull
+    public List<@FileExists File> getResourceConfigFiles()
+    {
+        return resourceConfigFiles;
+    }
+
+    @Config("kafka.config.resources")
+    @ConfigDescription("Optional config files")
+    public KafkaConfig setResourceConfigFiles(String files)
+    {
+        this.resourceConfigFiles = SPLITTER.splitToList(files).stream()
+                .map(File::new)
+                .collect(toImmutableList());
+        return this;
+    }
+
+    public KafkaConfig setResourceConfigFiles(List<File> files)
+    {
+        this.resourceConfigFiles = ImmutableList.copyOf(files);
+        return this;
+    }
+
+    public Properties getExternalResourceProperties()
```

Comment:
Please move this method out of Config class to some utility class.

Reply:
I created a ConfigurationUtils class similarly to Hive's one.